css/src/to-be-removed/admin-global.scss

```diff
@@ -1,8 +1,7 @@
 @import "menu";
```

Comment:
Within this file there are some pieces of CSS dedicated to the admin-pages. Please try to remove them

(ie, search for `wpseo-admin-page`, / `admin-page`)